packages/shorebird_cli/pubspec.yaml

```diff
@@ -9,7 +9,15 @@ environment:
   sdk: ">=3.0.0 <4.0.0"
 
 dependencies:
-  archive: ^3.3.6
+  # We're using a specific commit because the latest published version (3.4.10)
+  # does not include a sync close API for ZipFileEncoder.
+  # https://github.com/brendan-duncan/archive/commit/94bf62410e9a073451bda0d851a07e8b8863bde4
+  # This was causing a flaky test on windows when using `shorebird preview`
+  # https://github.com/shorebirdtech/shorebird/issues/1902
+  archive:
+    git:
+      url: https://github.com/brendan-duncan/archive
```

Comment:
Maybe leave a TODO here noting that we should switch away from a specific commit once a release containing the API we need is published